### PR TITLE
CRAN submission 0.18.2

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

**Review by strengejacke — COMMENTED** [review]
The update requires a confirmation from @LudvigOlsen that he is prepared to submit an update of _cvms_ (see https://github.com/LudvigOlsen/cvms/pull/28)

**Comment by strengejacke** [comment]
submitted

**Comment by strengejacke** [comment]
@IndrajeetPatil could you add a short review (like confirm), so I can see whether I can merge from VS Code? Currently, there's no merge button due to branch protection, and I'd like to know if this changes after a positive review.

File: cran-comments.md [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
+We checked all reverse dependencies, comparing R CMD check results across CRAN
 and dev versions of this package.
 
- * We saw 1 new problems: *effectsize*
- * We failed to check 0 packages
+ * We saw 1 new problems: *cvms*
  
-We have already informed the maintainer of this breaking change and the new version of *effectsize* on GitHub should soon be submitted to CRAN.
\ No newline at end of file
+We have already informed the maintainer of this breaking change and he is prepared to submit a new version of *cvms* to CRAN once *parameters* update is accepted on CRAN.
```
Comment (IndrajeetPatil) [inline cran-comments.md:8]: Looks good!

You can also cc Ludvig when you need to confirm that the revdepcheck issues are expected and the maintainer has been informed.

You might also want to mention that *statsExpressions* will also be broken, but this is due to _datawizard_ update, and unrelated to the current submission.

**Review by IndrajeetPatil — APPROVED** [review]

**Comment by strengejacke** [comment]
Ups, just submitted, so the CRAN comment did not mention statsExpression ;-) But I get an email anyway, will mention then. And, merging inside VS Code is now possible. Not sure whether I can "force merge" (will try for one of the other open submission PRs).

**Pull request merged**